tinker/faculty_bios/faculty_bio_controller.py

```diff
@@ -122,21 +124,160 @@ def _iterate_child_xml(self, child, author):
             school_array = []
             for school in child.findall('.//job-titles/school'):
                 school_array.append(school.text or 'Other')
+            if csv:
 
-            page_values = {
-                'author': child.find('author') or None,
-                'id': child.attrib['id'] or "",
-                'title': child.find('title').text or None,
-                'created-on': child.find('created-on').text or None,
-                'path': 'https://www.bethel.edu' + child.find('path').text or "",
-                'schools': school_array,
-                'last-name': child.find('.//last').text or None,
-                'deactivated': child.find('.//deactivate').text or None
-            }
+                return self.csv_file(child)
+
+            else:
+                page_values = {
+                    'author': child.find('author') or None,
+                    'id': child.attrib['id'] or "",
+                    'title': child.find('title').text or None,
+                    'created-on': child.find('created-on').text or None,
+                    'path': 'https://www.bethel.edu' + child.find('path').text or "",
+                    'schools': school_array,
+                    'last-name': child.find('.//last').text or None,
+                    'deactivated': child.find('.//deactivate').text or None
+                }
+
+            # Returns the page_values
             return page_values
         else:
             return None
 
+    def csv_file(self, child):
+
+        # Initial dictionary values
+        page_values = {
+            'first': child.find('.//first').text,
+            'last': child.find('.//last').text,
+            'highlight': '',
+            'email': '',
+            'started-at-bethel': '',
+            'research-interests': '',
+            'areas': '',
+            'teaching-specialty': '',
+            'quote': '',
+            'website': ''
+        }
+
+        # Checks if there is data in each field, if not puts in an empty string
+        if child.find('.//highlight').text is not None:
```

Comment:
these if's are very repetitive. You could consider creating a method:

```
def method(key):
    text = child.find('.//' + key).text
    if text:
          return text
    else:
          return ''
```

Also, is it a problem to return None or ''?

Reply:
Done, can't return None but can return ''

Reply:
Nice 👍